tests/fixtures/gcs_invalid_policy_response.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<Error>
+    <Code>InvalidPolicyDocument</Code>
+    <Message>The content of the form does not meet the conditions specified in the policy document.</Message>
+    <Details>Invalid exact match name: x-amz-storage-class</Details>
+</Error>
```

Comment:
#nitpick - newline at end of file missing

Reply:
fixed - 3bd3821a3470c153127e87f5ee6073a7f3b6fda3